spring-cloud-aws-samples/spring-cloud-aws-sqs-sample/src/main/java/io/awspring/cloud/sqs/sample/SpringSQSManualAck.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2013-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.awspring.cloud.sqs.sample;
+
+import io.awspring.cloud.sqs.listener.SqsMessageListenerContainer;
+import io.awspring.cloud.sqs.listener.acknowledgement.Acknowledgement;
+import io.awspring.cloud.sqs.listener.acknowledgement.AcknowledgementResultCallback;
+import io.awspring.cloud.sqs.listener.acknowledgement.handler.AcknowledgementMode;
+import io.awspring.cloud.sqs.operations.SqsTemplate;
+import io.awspring.cloud.sqs.operations.TemplateAcknowledgementMode;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.messaging.Message;
+import software.amazon.awssdk.services.sqs.SqsAsyncClient;
+
+import java.util.Collection;
+import java.util.UUID;
+
+@Configuration
+public class SpringSQSManualAck {
+
+	public static final String NEW_USER_QUEUE = "new-user-queue";
+
+	// Change to 'true' if you want to not ack.
+	private static final Boolean ACK = false;
+
+	private static final Logger LOGGER = LoggerFactory.getLogger(SpringSQSManualAck.class);
+
+	@Bean
+	public ApplicationRunner sendMessageToQueue(SqsTemplate sqsTemplate) {
+		LOGGER.info("Sending message");
+		return args -> sqsTemplate.sendAsync(to -> to.queue(NEW_USER_QUEUE)
```

Comment:
We can use the sync send method here - this way if an error occurs it will throw instead of returning a failed future.